bin/README.md

```diff
@@ -128,12 +128,20 @@ RootProjectDir
 You will need to include a `settings.sh` file in your top level `./openshift` directory that contains your project specific settings.
 
 At a minimum this file should contain definitions for your `PROJECT_NAMESPACE`, `GIT_URI`, and `GIT_REF` all of which should be setup to be overridable.
-
+The commented out sections are used for creating pull secrets, if you plan on using different docker registry, follow the instructions for setting up
+pull credentials [here](#setting-up-pull-secrets)
 **For Example:**
 ```
 export PROJECT_NAMESPACE=${PROJECT_NAMESPACE:-devex-von-permitify}
 export GIT_URI=${GIT_URI:-"https://github.com/bcgov/permitify.git"}
 export GIT_REF=${GIT_REF:-"master"}
+# export USE_PULL_CREDS=true
+# export PULL_CREDS=artifactory-creds
+# export CRED_SEARCH_NAME=artifacts-default
+# export DOCKER_REG=docker-remote.artifacts.developer.gov.bc.ca
+# export DOCKER_USERNAME=username
```

Comment:
Do not recommend credentials be included in the source code.  The example should show them being read from injected variables, for example `export DOCKER_USERNAME=${DOCKER_USERNAME}`.

Reply:
removed suggestion from readme. Injecting from param files is supported. For now the default is just to prompt the user for them when creating a secret

Reply:
I think you may be looking at an outdated readme somehow. The latest README in bin does not have any reference to DOCKER_USERNAME or DOCKER_PASSWORD

Reply:
okay weird, the REAME wasn't that updating and git wasn't detecting any chnages. I updated it manually and fixed it now